locale/messages.pot

```diff
@@ -664,119 +664,119 @@ msgstr ""
 msgid "NUCLEUS"
 msgstr ""
 
-#: ../src/auto-evo/AutoEvoRun.cs:119
+#: ../src/auto-evo/AutoEvoRun.cs:158
```

Comment:
It looks like you have the same thing going on as @Kasterisk with wrong line numbers appearing here.
Checking AutoEvoRun.cs file, this string *is* on line 119.

Reply:
Could it be babel version difference? I have `pybabel 2.9.0`

Reply:
I have that version too. All I did was use pip to install Babel and Babel-Thrive then add the folder with the executables to my PATH

Reply:
I can't find anything when googling. Maybe we should open a new issue to pybabel that we see line number disparencies on different systems? https://github.com/python-babel/babel

Reply:
Maybe I could rerun the ruby it to see if there are any differences?

Reply:
I'm pretty sure that won't do anything, but then again it will only take like 5 seconds to do.
Why I think that is the case that you and kasterisk both get the same line numbers.

Reply:
Rerunning does nothing

Reply:
I also have the same pybabel version. Did some test with running the script on master branch (clean, no changes made) and it still did the same with the line numbers, very strange.

Reply:
Opened an issue about this: https://github.com/python-babel/babel/issues/770